app/core/security.py

```
@@ -0,0 +1,45 @@
+"""
+보안 관련 유틸리티
+- JWT 토큰 생성/검증
+- 비밀번호 해싱/검증
+"""
+from datetime import datetime, timedelta
+from typing import Optional, Dict, Any
```

Ruff (0.14.11)

`typing.Dict` is deprecated, use `dict` instead

(UP035)